Extensions/3D/JsExtension.js

```diff
@@ -1125,16 +1126,17 @@ module.exports = {
       .addObject(
         'Cube3DObject',
         _('3D Box'),
-        _('A 3D box.'),
+        _('A box with an image on each face'),
```

Comment:
I'm not sure about this it doesn't sounds really English.
```suggestion
        _('A box with images for each face'),
```

Reply:
"A box with images for each face" suggests that there are different images for each face of the box. => let's go for this 👍